diff --git a/src/net.c b/src/net.c
index 5780686..dfd8160 100644
--- a/src/net.c
+++ b/src/net.c
@@ -2315,8 +2315,6 @@ static void
 handle_response(coap_context_t *context, coap_session_t *session,
   coap_pdu_t *sent, coap_pdu_t *rcvd) {
 
-  coap_send_ack(session, rcvd);
-
   /* In a lossy context, the ACK of a separate response may have
    * been lost, so we need to stop retransmitting requests with the
    * same token.
